documents/postman/README.md

```diff
@@ -0,0 +1,59 @@
+# Postman
+
+To download Postman, go [here](https://www.postman.com/downloads/). Postman allows you to send requests.
+
+## Postman collection
+
+The intention of this collection is to provide quick, easy functionality to send email, sms, and application notifications from non-admin users. As a non-admin user one can send a notification and get information regarding the notification. Creation, viewing and editing of templates should be done in the portal and not through the API.  
+
+The postman scripts use the environment variables and populate or update them as the scripts are executed.
+
+The basic development environment variables are in this folder which you can import along with the scripts. 
+
+## basic environment variables
+
+These environment variables should be defined before you can execute any of the scripts
+- notification-api-url: `{environment}-api.va.gov/vanotify`
```

Comment:
We have private and public URLs, you're only listing the public one. Is there value in listing this since it will come pre-populated?

Reply:
This is a good point, and if we do mention them we should mention both, and let the user know that `public` is for outside the VA network and `private` is for inside the VA network.

Reply:
@edDocMe360 Evan made a good point, can you add a comment that notification-api-url is publicly accessible and the private one can be accessed behind the VA Network for clarity?